src/Spaces/dss_cuda.jl

```diff
@@ -468,13 +468,13 @@ function load_from_recv_buffer!(::ClimaComms.CUDADevice, dss_buffer::DSSBuffer)
     if nrecv > 0
         nitems = nrecv * nlevels * nfid
         nthreads, nblocks = _configure_threadblock(nitems)
-        CUDA.synchronize()
+        CUDA.synchronize(;blocking=true)
         @cuda threads = (nthreads) blocks = (nblocks) load_from_recv_buffer_kernel!(
             pperimeter_data,
             recv_data,
             recv_buf_idx,
         )
-        CUDA.synchronize()
+        CUDA.synchronize(;blocking=true)
```

Comment:
And this one shouldn't be needed either?

Reply:
Same here. Generally speaking this should not be required (as with CPU MPI).